src/disco/store.js

```diff
@@ -30,6 +31,7 @@ export default function createStore({
       infoDialog,
       redirectTo,
       routing,
+      survey,
```

Comment:
Why do we need this for the disco pane? Are we showing the survey link there as well?

Reply:
It's because the server reads the cookie and dispatches an action for all requests. I could wrap those calls in an if-amo statement but it would add more complexity and possibly introduce bugs in Discopane since the code path would be different. With this approach, the two apps are more in sync which means possibly less bugs.

I actually dislike the way Discopane runs with a limited set of reducers. It feels like a premature optimization. All it does for us right now is trigger potential bugs.